vagrant/provisioning/roles/arkcase-ldap-config/tasks/main.yml

```diff
@@ -30,4 +30,22 @@
     - src: "spring-config-user-ad.properties"
       dest: "/home/arkcase/.arkcase/acm/spring/spring-config-user-{{ new_config_id }}-ad.properties"
     - src: "spring-config-user-ad.xml"
-      dest: "/home/arkcase/.arkcase/acm/spring/spring-config-user-{{ new_config_id }}-ad.xml"
\ No newline at end of file
+      dest: "/home/arkcase/.arkcase/acm/spring/spring-config-user-{{ new_config_id }}-ad.xml"
+  when: arkcase_version != "" and arkcase_version is version('3.3.7', '<')
+
+- name: ensure config server ldap folder exists
+  become: yes
+  become_user: arkcase
+  file:
+    path: /home/arkcase/.arkcase/acm/acm-config-server-repo/ldap
+    state: directory
+  when: arkcase_version != "" and arkcase_version is version('3.3.7', '>=')
```

Comment:
Probably should handle the case when `arkcase_version == ""`, like when deploying the current snapshot.

Reply:
Thanks for noticing this